charts/catalog/README.md

```diff
@@ -0,0 +1,68 @@
+# Service Catalog
+
+Service Catalog is a Kubernetes Incubator project that provides a
+Kubernetes-native workflow for integrating with [Open Service Brokers]
+(https://www.openservicebrokerapi.org/) to provision and bind to application
+dependencies like databases, object storage, message-oriented middleware, and
+more.
+
+For more information, [visit the project on github]
+(https://github.com/kubernetes-incubator/service-catalog).
+
+## Prerequisites
+
+- Kubernetes 1.6+ with Beta APIs enabled
+
+## Installing the Chart
+
+To install the chart with the release name `catalog`:
+
+```bash
+$ helm install charts/catalog --name catalog --namespace catalog
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `catalog` deployment:
+
+```bash
+$ helm delete catalog
```

Comment:
Did it not delete? Or was the problem that you couldn't re-install using the same release name? If the latter, I have encountered that too, and I believe it's a Helm bug.